README.md

```diff
@@ -116,7 +116,7 @@ export He_Cosmos_MoviesCol=movies
 az cosmosdb create -g $He_Cosmos_RG -n $He_Name
 
 # create the database
-az cosmosdb sql database create -d $He_Cosmos_DB -g $He_Cosmos_RG -n $He_Name
+az cosmosdb sql database create -a $He_Name --n $He_Cosmos_DB -g $He_Cosmos_RG
```

Comment:
is "--n" correct? should it be -n or --name?

Reply:
Weirdness, I originally tested it with full monikers and then removed to shorten them. Tested it again to make sure that I did not make a typo somewhere but apparently CLI will accept the double dash (--n). That being said, keeping it that way does not make sense. I will send update the PR with a single dash. Also will update the formatting as per spec.